tests/keras/metrics_training_test.py

```diff
@@ -81,6 +82,7 @@ def test_sensitivity_metrics():
 
 
 @pytest.mark.skipif(K.backend() != 'tensorflow', reason='requires tensorflow')
```

Comment:
I believe that we have too much problem with this tests for what it's worth. Can you disable it? We'll enable it again later once we find a fix.
```suggestion
@pytest.mark.skipif(True, reason='It is a flaky test, see #13477 for more context.')
```

Reply:
OK I update it right now, Thanks.